dotnet/samples/06.assistants.a.mathBot/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Teams.AI;
+using Microsoft.Teams.AI.AI;
+using Microsoft.Teams.AI.AI.Planners;
+using MathBot;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+builder.Services.AddHttpClient("WebClient", client => client.Timeout = TimeSpan.FromSeconds(600));
+builder.Services.AddHttpContextAccessor();
+
+// Load configuration
+var config = builder.Configuration.Get<ConfigOptions>()!;
+if (config.OpenAI == null || string.IsNullOrEmpty(config.OpenAI.ApiKey))
+{
+    throw new Exception("Missing OpenAI configuration.");
+}
+
+// Missing Assistant ID, create new Assistant
+if (string.IsNullOrEmpty(config.OpenAI.AssistantId))
+{
+    Console.WriteLine("No Assistant ID configured, creating new Assistant...");
+    string newAssistantId = AssistantsPlanner<AssistantsState>.CreateAssistantAsync(config.OpenAI.ApiKey, null, new()
+    {
+        Name = "Math Tutor",
+        Instructions = "You are a personal math tutor. Write and run code to answer math questions.",
+        Tools = new()
+        {
+            new()
+            {
+                Type = "code_interpreter"
```

Comment:
Just noticed that openai js sdk wraps the type: https://github.com/openai/openai-node/blob/master/src/resources/beta/assistants/assistants.ts#L308. Shall we do the same thing for easy and accurate use?

Reply:
Good idea.
I'll add shortcuts like `Tool.Code()`, `Tool.Function()` in coming PR.